scripts/west_commands/ncs_ironside_se_update.py

```diff
@@ -88,8 +88,19 @@ def do_run(self, args, unknown_args):
         def nrfutil_device(cmd: str) -> str:
             cmd = f"nrfutil device {cmd} --serial-number {args.serial}"
             self.dbg(cmd)
+
             result = subprocess.run(cmd, shell=True, text=True, capture_output=True)
+
             self.dbg(result.stdout)
+
+            if result.returncode != 0:
+                # Always show error messages, even in non-verbose mode
+                self.err(f"Command failed with return code {result.returncode}: {cmd}")
+                if result.stderr.strip():
+                    self.err(f"Error output: {result.stderr.strip()}")
+
+                raise RuntimeError(f"nrfutil command failed: {cmd}")
```

Comment:
Could we avoid printing the failing command twice, maybe by moving the return code info here?

Reply:
fixed